woocommerce/woocommerce-ios
# Fix buildkite test annotation message

## Description
A follow-up PR on the messaging on the Buildkite logs based on comments in https://github.com/woocommerce/woocommerce-ios/pull/9726

The messaging changes include the following:
1. Revert the previous removal of auto-expanding the "Testing" section in Buildkite logs when there are failures
2. Updated the message to be similar to the one printed on WP/JPiOS but with "step" updated to "section."

## Testing instructions
As this is only a messaging update, as long as CI is green then the build should be good.
